packaging_automation/update_pgxn.py

```diff
@@ -32,26 +35,33 @@ def update_pgxn_files(project_version: str, template_path: str, exec_path: str):
     parser = argparse.ArgumentParser()
     parser.add_argument('--prj_ver', required=True)
     parser.add_argument('--gh_token', required=True)
+    parser.add_argument("--pipeline", action="store_true")
+    parser.add_argument('--exec_path')
     parser.add_argument('--is_test', action="store_true")
     args = parser.parse_args()
 
-    execution_path = f"{os.getcwd()}/{CHECKOUT_DIR}"
     github_token = args.gh_token
     main_branch = "pgxn-citus"
 
-    tools_path = os.getcwd()
-    initialize_env(execution_path, PROJECT_NAME, CHECKOUT_DIR)
+    if args.pipeline:
+        if not args.exec_path:
+            raise ValueError(f"exec_path should be defined")
+        execution_path = args.exec_path
+    else:
+        execution_path = f"{os.getcwd()}/{CHECKOUT_DIR}"
+        initialize_env(execution_path, PROJECT_NAME, execution_path)
+
     os.chdir(execution_path)
 
     run(f"git checkout {main_branch}")
     pr_branch = f"pgxn-citus-push-{args.prj_ver}-{uuid.uuid4()}"
     run(f"git checkout -b {pr_branch}")
-    template_path = f"{tools_path}/packaging_automation/templates/pgxn"
+    template_path = f"{BASE_PATH}/templates/pgxn"
     update_pgxn_files(args.prj_ver, template_path, execution_path)
     commit_message = f"Bump pgxn to version {args.prj_ver}"
     run(f'git commit -a -m "{commit_message}"')
     if not args.is_test:
         run(f'git push --set-upstream origin {pr_branch}')
-
-    if not args.is_test:
         create_pr(github_token, pr_branch, commit_message, REPO_OWNER, PROJECT_NAME, main_branch)
```

Comment:
It would be a nice touch if we open draft PRs instead of normal ones. This can make things easy for 2 round of reviews: first by @gurkanindibay who will mark the PR as ready, then another person on the team who will approve it.